dockerfiles/scripts/checkout_submodules.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash 
+
+echo "checking out submodule versions"
+
+CUR_PWD=$(pwd)
+cd onnxruntime/cmake/external
+
+# Point to correct onnx-tensorrt
+echo "$1"
+if [ ! "$1" = "8.2" ]
+then
+    cd onnx-tensorrt
+    git remote update
+    if [ "$1" = "8.0" ]
+    then 
+        git checkout "$1"'-GA'
+    fi
+    if [ "$1" = "7.2" ]
+    then 
+        git checkout "$1"'.1'
+    fi 
+    cd ..
+fi
+
+# Point to correct protobuf
+PROTO_VER=$(apt list | grep -oP -m 1 'protobuf-compiler.* \K3[\.\d]+')
+echo "$PROTO_VER"
+
+if awk 'BEGIN{exit !(ARGV[1]<ARGV[2])}' "$PROTO_VER" "3.11.0"
```

Comment:
It's difficult to compare decimals in shell script. I do think this isn't the best way but am struggling to find alternatives.